mitodl/open-discussions
# add pin post api

#### What are the relevant tickets?

closes #194 

#### What's this PR do?

this adds a field ('stickied') to the Post API which can be toggled to make a post 'pinned' / 'sticky'.

#### How should this be manually tested?

You should be a moderator on your channels, so you should be able to toggle the boolean using the DRF web interface (just type `{"stickied": true}` and make sure it saves).
